.gitignore

```diff
@@ -8,3 +8,6 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
+# Remove all the *.wasm files
+*.wasm
```

Comment:
Oh, can you revert this change? We want to keep checking in profiler_get_symbols_bg.wasm.